.gitignore

```diff
@@ -24,3 +24,6 @@ venv
 
 # python win libs
 **.pyd
+
+#config.yml
+./etc/config/config.yml
```

Comment:
The config.yml file is already being ignored in line 8, why did you add it again?

Reply:
That is a good question! Removing.